oauth2client/service_account.py

```diff
@@ -145,14 +152,20 @@ def _to_json(self, strip, to_serialize=None):
             strip, to_serialize=to_serialize)
 
     @classmethod
-    def _from_parsed_json_keyfile(cls, keyfile_dict, scopes):
+    def _from_parsed_json_keyfile(cls, keyfile_dict, scopes,
+                                  token_uri=GOOGLE_TOKEN_URI,
```

Comment:
Optional keyword parameters in private behaviors always seem weird to me because all the call sites are completely under the author's control. Should these really be optional? At how many call sites are they passed? At how many call sites are they not passed?